### PR TITLE
Phoebe/farm profile page

### PR DESCRIPTION
**Pull request by phoebeli23**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

Created Farm Profile Page (for farms participating in Food Hub)
Accessible at route localhost:3000/farm/:id
https://www.notion.so/77891b5a221c435ba97bdff45d6adb39?v=eba74d9aeb7b447a898462fe7b5222c2&p=5692bdbb85644e8c9596c8710fb94947

### How to Review

New Files:
1. FarmProfile.js
2. FarmContactCard.js
3. FarmProfile.css

FarmProfile.js is the farm profile page component, which renders FarmContactCard.js
Both files use FarmProfile.css for styling 

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

Pipe in real data
Optimize CSS: align text and icons, make responsive

### Screenshots

farmProfilePageScreenshot

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

yay css is hard

CC: @dfangshuo

### CODE REVIEW

File: src/styles/FarmProfile.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.farm-contact-card__main__text__details {
+  margin-top: 20px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.282191px;
+  color: rgba(51, 51, 51, 0.5);
```
Comment (aliceyezhao) [inline src/styles/FarmProfile.css:101]: We should make a file of color hex codes and import them for consistency.

File: src/styles/FarmProfile.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  font-family: 'Lora', serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 36px;
+  letter-spacing: 1px;
+}
+
+.farm-profile__back-button {
+  font-family: 'Inter', sans-serif;
```
Comment (aliceyezhao) [inline src/styles/FarmProfile.css:20]: We should also change fonts globally in App.css to save some lines. I kept the default fonts for now

File: src/screens/farmProfile/components/FarmContactCard.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import farmProfilePhoto from '../../../assets/farmProfilePhoto.png';
+import checkIcon from '../../../assets/checkIcon.png';
+import cancelIcon from '../../../assets/cancelIcon.png';
+import userIcon from '../../../assets/userIcon.png';
+
+import '../../../styles/FarmProfile.css';
+
+class FarmContactCard extends React.PureComponent {
+  render() {
+    // const { id, farmName, farmerName, phone, email, address, location, inspector } = this.props;
```
Comment (aliceyezhao) [inline src/screens/farmProfile/components/FarmContactCard.js]: delete

File: src/screens/farmProfile/FarmProfile.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null
+    };
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    const { id } = match.params;
```
Comment (aliceyezhao) [inline src/screens/farmProfile/FarmProfile.js]: rename to `farmId`

File: src/screens/farmProfile/FarmProfile.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+class FarmProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null
+    };
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
```
Comment (aliceyezhao) [inline src/screens/farmProfile/FarmProfile.js:15]: `match` is extracting the farmID from the route URL right? Maybe rename it unless if this is convention.

File: src/screens/farmProfile/FarmProfile.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+        <div className="farm-profile__body">
+          <div className="farm-profile__body__left">
+            <FarmContactCard
+              id="123"
+              farmerName="Farmer Name"
+              phone="(xxx) - xxx - xxxx"
+              email="farmerfarmer@farmer.com"
+              address="xxxxx Farmer Lane Kauai, HI, xxxxx"
+              gapCertified
```
Comment (aliceyezhao) [inline src/screens/farmProfile/FarmProfile.js:41]: should you set `gapCertified`?

**Review by aliceyezhao — CHANGES_REQUESTED** [review]
Just some convention stuff which has to do with other stuff that we need to discuss as a team as well. Great job with this! 

Also, remove your yarn.lock file from git since it is autogenerated locally

**Review by andi-halim — APPROVED** [review]
looks goooood!
- bottom card leaks out of the largest container so that should be fixed on the next PR. 
- good job with all the props I forgot to do that with my pr wops

File: src/screens/farmProfile/components/FarmContactCard.js [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                      src={checkIcon}
+                      alt="certification"
+                    />
+                    {`GAP Certified since ${certificationDate}`}
+                  </div>
+                ) : (
+                  <div>
+                    <img src={cancelIcon} alt="certification" />
+                    Not GAP Certified
+                  </div>
```
Comment (aliceyezhao) [inline src/screens/farmProfile/components/FarmContactCard.js:57]: @halimousine this is an example of conditional rendering

**Review by aliceyezhao — APPROVED** [review]

**Review by niwong — COMMENTED** [review]
Late to the party but this is some beautiful code. Nice job @phoebeli23 and thank you so much @aliceyezhao for reviewing and leaving insightful comments. Looking great, team.

**Pull request merged**